src/utils/helpers/list/useGetFilterDefinitions.ts

```diff
@@ -109,19 +110,25 @@ export const useApplicationFilters = () => {
     },
     stage: {
       type: 'staticList',
+      default: defaultFilters.some((filter) => filter === 'stage'),
+      visibleTo: checkFilterVisibility("STAGE"),
       title: strings.FILTER_STAGE,
       options: {
         getListQuery: constructFilterListQuery(getStageFilterList, 'applicationListFilterStage'),
       },
     },
     search: {
       type: 'search',
+      default: defaultFilters.some((filter) => filter === 'search'),
+      visibleTo: ['applicant' as USER_ROLES, 'reviewer' as USER_ROLES], // Always Visible 
```

Comment:
```suggestion
      visibleTo: [USER_ROLES.APPLICANT, USER_ROLES.REVIEWER], // Always Visible
```

Reply:
Also, personally I like this definition way more. I think it would be better if they were all like this, I don't see why we need a whole extra map and function. This is super readable and clear, why complicate it?

Reply:
Oh, totally agreed!